code/modules/organs/organ_external.dm

```diff
@@ -137,53 +180,32 @@
 				burn = max(0, burn - can_inflict)
 		//If there are still hurties to dispense
 		if (burn || brute)
-			if (status & ORGAN_ROBOT && body_part != UPPER_TORSO && body_part != LOWER_TORSO)
-				droplimb(1) //Robot limbs just kinda fail at full damage.
-			else
-				//List organs we can pass it to
-				var/list/datum/organ/external/possible_points = list()
-				if(parent)
-					possible_points += parent
-				if(children)
-					possible_points += children
-				if(forbidden_limbs.len)
-					possible_points -= forbidden_limbs
-				if(possible_points.len)
-					//And pass the pain around
-					var/datum/organ/external/target = pick(possible_points)
-					target.take_damage(brute, burn, sharp, edge, used_weapon, forbidden_limbs + src)
+			owner.shock_stage += brute+burn
```

Comment:
I'm still strongly against damage transfering up to parent, players will learn to switch up their attacks, and it will fix armor and helmets being useless. Can we at least try it this way first?